content/en/design/guides/mapping-hierarchy.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Mapping User on the CHT Hierarchy"
+linkTitle: Mapping Users
+weight: 3
+description: >
+  Mapping users on the CHT hierarchy
+relatedContent: >
+
+---
+
+## Purpose Of the Guide
+
+The tutorial will take you through mapping of users on CHT hierarchy, including:
+
+- Defining the hierarchy (reporting structure)
+- Defining user roles
+- Mapping user personas to the CHT hierarchy
+
+## Brief Overview of Key Concepts
+
+User persona is a generalized character that embodies a particular type of user.
+
+User roles are the activities that the user personas are expected to carry out.
+
+Hierarchy represents the reporting structures
+
+## Prerequisites/Required Resources
+
+User personas profiles
```

Comment:
Link to persona page

Reply:
Done. I also added a `see also` that links to the other page on Personas.